examples/config-dev.yaml

```diff
@@ -42,32 +42,65 @@ telemetry:
 #   level: "debug"
 #   format: "text" # can also be "json"
 
-# Default values shown below
```

Comment:
Could this be left out, please? Improvements to the example config are of course welcome, but in a separate PR, please 😃